devel/revive-hf.lua

```diff
@@ -0,0 +1,40 @@
+-- Revives the specified historical figure
+
+--[====[
+
+devel/revive-hf
+=============
+
+Revivs the specified historical figure, even if off-site.
+
+Usage::
+
+    devel/revive-hf HISTFIG_ID
+
+Arguments:
+
+``histfig_id``:
+    the ID of the historical figure to target
+
+]====]
+
+local target_hf = -1
+
+for _, arg in ipairs({...}) do
+    if tonumber(arg) and target_hf == -1 then
+        target_hf = tonumber(arg)
+    else
+        qerror('unrecognized argument: ' .. arg)
+    end
+end
```

Comment:
Since you only interpret a single argument anyway, this could be simplified down to something like:
```
local args = {...}
local target_hf = tonumber(args[1])
if not target_hf then qerror('please pass the ID of the target historical figure') end
```

Reply:
unresolving since this comment still looks relevant

Reply:
Resolved.